service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/ThirdPartyApplicationContentProvider.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.notify.sendtogatekeeper.ManagedLATemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.SharedNotifyContentProvider;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ThirdPartyApplicationContentProvider extends SharedNotifyContentProvider {
```

Comment:
Usually we have consistency about handler name and  provider name and returned template (btw, dont know why. we call them templates, they are customizations, params, personalizations, but not templates)